### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by lukicsl**
wget http://central.maven.org/maven2.. need  to changed to wget http://search.maven.org/maven2...

### CODE REVIEW

**Comment by openhab-5iver** [comment]
Why search rather than direct? Is something not working?

**Comment by lukicsl** [comment]
central.maven.org is not valid
image

File: Docker/Python/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ ENV \
 ## Install to /opt but allow writing $class files by giving every one permission to write, openhab user is created only after container has started
 RUN \
   mkdir -p ${JYTHON_HOME} && \
-  wget http://central.maven.org/maven2/org/python/jython-installer/${JYTHON_VERSION}/jython-installer-${JYTHON_VERSION}.jar -O /tmp/jython-installer.jar && \
+  wget http://search.maven.org/maven2/org/python/jython-installer/${JYTHON_VERSION}/jython-installer-${JYTHON_VERSION}.jar -O /tmp/jython-installer.jar && \
```
Comment (openhab-5iver) [inline Docker/Python/Dockerfile]: No need for a search... use a direct link...
```
wget https://repo1.maven.org/maven2/org/python/jython-installer/${JYTHON_VERSION}/jython-installer-${JYTHON_VERSION}.jar -O /tmp/jython-installer.jar && \
```

File: Docker/Python/Dockerfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ ENV \
 ## Install to /opt but allow writing $class files by giving every one permission to write, openhab user is created only after container has started
 RUN \
   mkdir -p ${JYTHON_HOME} && \
-  wget http://central.maven.org/maven2/org/python/jython-installer/${JYTHON_VERSION}/jython-installer-${JYTHON_VERSION}.jar -O /tmp/jython-installer.jar && \
+  wget http://repo1.maven.org/maven2/org/python/jython-installer/${JYTHON_VERSION}/jython-installer-${JYTHON_VERSION}.jar -O /tmp/jython-installer.jar && \
```
Comment (openhab-5iver) [inline Docker/Python/Dockerfile]: One more... I forgot that Maven Central requires https...

```suggestion
  wget https://repo1.maven.org/maven2/org/python/jython-installer/${JYTHON_VERSION}/jython-installer-${JYTHON_VERSION}.jar -O /tmp/jython-installer.jar && \
```

**Comment by openhab-5iver** [comment]
Thank you! Next time please sign your commits... https://www.openhab.org/docs/developer/contributing.html#sign-your-work.

**Pull request merged**